traits/has_traits.py

```diff
@@ -129,7 +128,7 @@ def ViewElements():
     StaticTraitChangeNotifyWrapper,
 )
 
-if sys.version_info[0] >= 3:
+if six.PY3:
```

Comment:
Please can you reverse the sense of the `if` here so that we can use `if six.PY2`. `if six.PY3` has the problem that it'll do the wrong thing if/when Python 4.0 is released (which could happen at some point).

Reply:
Let's be explicit and use `if/elif` instead.

Reply:
nvm. implementing your recommendation.